astropy/coordinates/transformations/__init__.py

```diff
@@ -0,0 +1,32 @@
+"""Coordinate transformations."""
+
+from astropy.coordinates.transformations.affine import (
```

Comment:
I think a submodule is one place where local imports are the right way to go.

Reply:
✅